lib/install/config/loaders/installers/elm.js

```diff
@@ -4,11 +4,11 @@ const { env } = require('../configuration.js')
 const elmSource = path.resolve(process.cwd())
 
 const loaderOptions = () => {
-    if (env.NODE_ENV === "production") {
-        return `elm-webpack-loader?cwd=${elmSource}`
-    }
+  if (env.NODE_ENV === 'production') {
```

Comment:
@bkuhlmann It's just an ENV flag so, it could be toggled in any environment. `NODE_ENV=production ./bin/webpack` so, don't think we need to be that explicit